tons of docs (#7)
xf8b/documentation: docs/xf8bot/commands/music/volume.md

---
layout: default
title: Volume Command
permalink: /xf8bot/commands/music/volume/
---
# Volume Command
## Info
The `volume` command changes the volume of the audio playing through the bot.

**Anyone can use this command.**

---

## Usage

* NOTE: Required Parameters = <> and Optional Parameters = []

Usage: `{prefix}volume <volume>`

> * NOTE: The max volume is 400 and min volume is 1.

Reviewer: leave a warning that it MAY blast your ears out lol

Reviewer: ```suggestion
* NOTE: The max volume is 400 and min volume is 1.
* WARNING: Setting a volume higher than 100 MAY blast your ears out. We are not liable for that, please be careful when using this.
```

Reviewer: ```suggestion
* NOTE: The max volume is 400 and min volume is 1.
* WARNING: Setting a volume higher than 100 **MAY** blast your ears out. We are not liable for that, please be careful when using this.
```

---
layout: default
title: Volume Command
permalink: /xf8bot/commands/music/volume/
---
# Volume Command
## Info

> The `volume` command changes the volume of the audio playing through the bot.

Reviewer: ```suggestion
The `volume` command changes the volume of the music currently playing. **Can blast out your ears if the volume is over 100, please be careful.**
```
